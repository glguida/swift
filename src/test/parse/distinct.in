distinct Ball a[10], b;