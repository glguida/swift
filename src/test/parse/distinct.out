(BlogProgram:
  (args:
    (DistinctDecl:
      :type Ball
      :var (:        (: a 10 )        (: b 1 )      ) )
    )
  )
)
